nest-cli.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
small typo in the above file's folder name `documenation => documentation`

Reply:
will rename it to `docs` to be similar to webframe